gui/build.py

```diff
@@ -134,6 +135,11 @@ def do_bundle(options):
     if IS_WIN:
         makensis_path = os.path.join(options.nsis_path, "makensis.exe")
         call(makensis_path, 'wininst.nsi', cwd='wininst')
+    elif IS_MAC:
+        arg = ['hdiutil', 'create', 'dist/mozregression-gui.dmg', '-srcfolder']
+        arg.append('dist/')
+        arg.append('-ov')
+        call(*arg)
```

Comment:
Nit: Since you are not dynamically changing args, you can just do something like:
```python
call('hdiutil', 'create', 'dist/mozregression-gui.dmg', '-srcfolder', 'dist/', '-ov')
```